luigi/lock.py

```diff
@@ -50,6 +50,8 @@ def getpcmd(pid):
                 spid, scmd = line.strip().split(' ', 1)
                 if int(spid) == int(pid):
                     return scmd
+    # Fallback instead of None, for e.g. Cygwin where -o is an "unknown option" for the ps command:
```

Comment:
I was more inclined to a a generic fallback, as enumerating the platforms just won't work as I experienced recently :
https://github.com/pybuilder/pybuilder/issues/328

Else this code should just stick to the POSIX `ps` argument `-f` to get a full output and do some parsing, and not rely on `-o`